### PR TITLE
Adding control to udf list

### PR DESCRIPTION
**Pull request by mayabrandi**
## Description
Contol is set in NIPT orders and should also be propagated to lims. The udf control has ben added to sample level in lims

### Added
- Control field in list of UDFs to set when creating a order.

## Review
- [ ] code approved by
- [x] tests executed by @mayabrandi 
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

**Comment by mayabrandi** [comment]
Skärmavbild 2021-11-24 kl  08 34 22

**Comment by mayabrandi** [comment]
@patrikgrenfeldt how do we test this? Can you test? The new field is added to sample on both lims stage and prod

File: cg/constants/lims.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "capture_kit": "Capture Library version",
     "collection_date": "Collection Date",
     "comment": "Comment",
+    "control": "control",
```
Comment (patrikgrenfeldt) [inline cg/constants/lims.py:7]: Seems to me like we usually name the LIMS UDF with a leading capital letter:
```suggestion
    "control": "Control",
```

**Comment by mayabrandi** [comment]
Tested locally. Control udf was set properly via order

Skärmavbild 2021-11-24 kl  14 55 43

Skärmavbild 2021-11-24 kl  14 57 32

Control udf was not set on saple with control=='' in orderform

Skärmavbild 2021-11-24 kl  14 57 16

.

**Review by patrikgrenfeldt — APPROVED** [review]
Looking good 👍

**Pull request merged**